mapclientplugins/geometryfitter/view/geometryfitterwidget.py

```diff
@@ -26,10 +25,9 @@ def field_is_managed_group_mesh(field, mesh):
     Chooser conditional function limiting to field group with a mesh group for mesh.
     """
     if field_is_managed_group(field):
-        elementGroup = field.castGroup().getFieldElementGroup(mesh)
-        if elementGroup.isValid():
-            if elementGroup.getMeshGroup().getSize() > 0:
-                return True
+        meshGroup = field.castGroup().getMeshGroup(mesh)
+        if meshGroup.isValid() and (meshGroup.getSize() > 0):
```

Comment:
Could this be `and not meshGroup.isEmpty()`?

Reply:
No. Mesh/MeshGroup only have the API getSize(). FieldGroup has itEmpty() which is a more complicated query over all subelement and subregion groups.

Reply:
Actually I think the size query is unnecessary; it's probably got something in it if it isValid(), so I'll remove. If it is in fact empty, it's the user's problem if they chose such a mesh group.
Removed check.